app/cache/cache-config.js

```diff
@@ -0,0 +1,8 @@
+const CacheService = require('./cache-service');
+const config = require('config');
+
+const userInfoCache = new CacheService('UserInfoCache', config.get('cache.user_info_ttl'), config.get('cache.user_info_check_period'));
```

Comment:
this was not part of the requirements, but I was thinking it would be probably good to have the ability to enable/disable caching via configuration

Reply:
Added new config option `cache.user_info_enabled` as discussed 👍